app/src/test/java/bisq/android/tests/util/DateUtilTest.kt

```diff
@@ -22,27 +22,44 @@ import com.google.gson.Gson
 import com.google.gson.JsonObject
 import org.junit.Assert.assertEquals
 import org.junit.Assert.assertNull
+import org.junit.Assume.assumeFalse
+import org.junit.Assume.assumeTrue
 import org.junit.Test
 import java.util.Date
 import java.util.Locale
+import java.util.TimeZone
 
 class DateUtilTest {
 
     private val dateUtil = DateUtil()
 
     @Test
     fun testDefaultFormatReturnsFormattedString() {
-        assertEquals("2022-05-08 01:46:43", DateUtil.format(1651974403000L))
+        assertEquals("2022-05-08 01:46:43", DateUtil.format(1651974403000L, timezone = TimeZone.getTimeZone("UTC")))
     }
 
     @Test
     fun testFormatWithSpecifiedLocaleReturnsFormattedString() {
-        assertEquals("2022-05-08 01:46:43", DateUtil.format(1651974403000L, Locale.GERMAN))
+        assertEquals("2022-05-08 01:46:43", DateUtil.format(1651974403000L, Locale.GERMAN, timezone = TimeZone.getTimeZone("UTC")))
```

Comment:
Since you are providing the timezone parameter in all your tests, you are not covering the case where if the timezone is not provided. I would think that is a case that should be covered since none of the actual production code specifies a timezone parameter.

Perhaps you can set the default timezone so that it is consistent for any tests that do not provide the timezone parameter? For example:
`TimeZone.setDefault(TimeZone.getTimeZone("UTC"))`

Reply:
I was trying to avoid `setDefault` to not mess with the JVM's default timezone.
The test methods look cleaner now though..